packages/react-scripts/template/src/components/Logo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import logo from './Logo.svg';
```

Comment:
```suggestion
import logo from './logo.svg';
```
Just to avoid potential capitalization issues on other platforms

Reply:
Yeah, I was back and forth here. I only capitalized it, because it was becoming coupled with `Logo.jsx`, and it's a best-practice to capitalize `Logo.jsx` along with things like `Logo.css`, etc. It made sense. I thought I capitalized the actual file name as well, but maybe that didn't come through in the PR?

Reply:
Will address in a separate PR